ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -781,14 +889,24 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
                         </Subheader>
                         {valueSet.values.items.map(domainValue =>
                           <ValueListItem data-test-id='value-list-items'
-                            key={domainValue.value} domainValue={domainValue}
+                            key={domainValue.value} domain={valueSet.domain} domainValue={domainValue}
                             onChange={() => this.selectDomainValue(valueSet.domain, domainValue)}
                             checked={fp.some({domain: valueSet.domain, value: domainValue.value},
                               selectedDomainValuePairs)}/>
                         )}
                       </div>)
                     }
                   </div>
+                  {valueSets.length > 0 && <FlexRow style={{
+                    width: '100%', height: '1.375rem', backgroundColor: colorWithWhiteness(colors.dark, 0.9),
+                    color: colors.primary, paddingLeft: '0.4rem', fontSize: '13px', lineHeight: '16px',
+                    alignItems: 'center'}}>
+                    <a href={'https://aousupporthelp.zendesk.com/hc/en-us/articles/' +
```

Comment:
This link does not work for me

Reply:
It shouldn't. I talked to Karthik, we don't have a zendesk for any environment but prod, and SSO will error out the site if you aren't coming from prod. This is problematic because it means we can't easily test these links from the site, but this link should work if you are coming from a web browser authed as a researchallofus account

Reply:
Link is correct when viewed as my Prod persona, thanks